src/main/kotlin/net/axay/kspigot/event/KSpigotListeners.kt

```diff
@@ -61,16 +61,18 @@ inline fun <reified T : Event> SingleListener<T>.register(
  * @param T the type of event to listen to
  * @param priority the priority when multiple listeners handle this event
  * @param ignoreCancelled if manual cancellation should be ignored
+ * @param register if the event should be registered
```

Comment:
if the event should be registered immediately*

Reply:
digga